tests/testthat/test-censor-probs.R

```diff
@@ -0,0 +1,65 @@
+
+test_that("reverse Kaplan-Meier curves", {
+  skip_if_not_installed("prodlim")
+  skip_if_not_installed("survival")
+  skip_if_not_installed("censored")
+  skip_if(utils::packageVersion("parsnip") < "1.0.3.9001")
+
+  library(survival)
+  library(censored)
+  library(prodlim)
+
+  lung <- lung[complete.cases(lung), ]
+
+  mod_fit <-
+    survival_reg() %>%
+    fit(Surv(time, status) ~ age + sex, data = lung)
+
+  expect_true(any(names(mod_fit) == "censor_probs"))
+  expect_true(
+    inherits(mod_fit$censor_probs,
+             c("censoring_model_reverse_km", "censoring_model")))
```

Comment:
this also could be a 
```suggestion
  expect_s3_class(
    mod_fit$censor_probs,
    c("censoring_model_reverse_km", "censoring_model")
  )
```